crypto/evp/m_sigver.c

```diff
@@ -31,6 +31,7 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
 {
     EVP_PKEY_CTX *locpctx = NULL;
     EVP_SIGNATURE *signature = NULL;
+    char locmdname[80] = "";     /* 80 chars should be enough */
```

Comment:
Could we have (perhaps internal) #define somewhere for this? Also below in the actual function implementation there is 100. Why this difference?

Reply:
I didn't remember between them...

A macro would be suitable.  How about an internal `OSSL_MAX_NAMESIZE`?

Reply:
That would be fine.

Reply:
We have those in a number of places, so I'll make a sweep and submit that in another PR.

Reply:
See #10928.  If that one gets a quick approval and merge, it'll be easy to rebase this one on top of that, and adapt accordingly.